### PR TITLE
Activate: Store creation unit tests

### PR DESCRIPTION
**Pull request by 0nko**
This PR adds the missing unit tests for the Plans and Installation screen.

**To test:**
Since the tests are run automatically on the CI, just make sure all tests pass.

### CODE REVIEW

**Comment by ParaskP7** [comment]
FYI @0nko @JorgeMucientes that I've just moved this issue's milestone to `11.8` as I'll be starting code freeze on `11.7` soon.

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/login/storecreation/PlansViewModelTest.kt [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        val observedPlan = captor.firstValue as? PlanState
+        assertNotNull(observedPlan)
+
+        assertEquals(planInfo.name, observedPlan.plan.name)
+        assertEquals(planInfo.formattedPrice, observedPlan.plan.formattedPrice)
+        assertEquals(planInfo.billingPeriod, observedPlan.plan.billingPeriod)
+    }
+
+    @Test
+    fun `when a site is created, a plan with is added to site's checkout cart and a WebView is launched`() =
```
Comment (JorgeMucientes) [inline WooCommerce/src/test/kotlin/com/woocommerce/android/ui/login/storecreation/PlansViewModelTest.kt]: Minor, missing something in test name? "`when a site is created, a plan with [What?] is added to site's...." with productId? With planPathSlug? With "the correct data" ?

**Review by JorgeMucientes — APPROVED** [review]

**Comment by JorgeMucientes** [comment]
Nice job @0nko, test code looks good! I just left a non-blocking nitpick in one of the tests naming.

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/login/storecreation/PlansViewModelTest.kt (hunk 1 above)
Comment (0nko) [reply WooCommerce/src/test/kotlin/com/woocommerce/android/ui/login/storecreation/PlansViewModelTest.kt]: Nice catch, `with` is extra there :)

**Pull request merged**